ocp_resources/resource.py

```diff
@@ -463,8 +463,10 @@ def _sigint_handler(self, signal_received, frame):
         self.__exit__(exception_type=None, exception_value=None, traceback=None)
         sys.exit(signal_received)
 
-    def deploy(self):
+    def deploy(self, wait=False):
```

Comment:
We have more resources that define wait, please add wait there as well.

Reply:
I will update those resources.